packages/argo-checkout/src/components/Banner/Banner.ts

```diff
@@ -0,0 +1,43 @@
+import {createRemoteComponent} from '@remote-ui/core';
+
+/**
+ * Success:
```

Comment:
We're missing the `info` description.

Something like that:

```suggestion
   * Info:
   * Use to convey general information or actions that aren’t critical or tied to a particular action. 
 
   * Success:
```